src/networks.ts

```diff
@@ -189,6 +190,14 @@ export const PRODUCTION_NETWORKS: { [chainId: number]: PublicNetwork } = {
     blockExplorer: "https://scrollscan.com",
     cctpDomain: CCTP_NO_DOMAIN,
   },
+  [CHAIN_IDs.SOLANA]: {
+    name: "Solana",
+    family: SOLANA,
```

Comment:
We might want something more generic here in case we end up supporting some hybrid SVM rollup onto an EVM chain or something.

```suggestion
    family: SVM,
```

Reply:
Yes, good point 👍

Reply:
[fixup](https://github.com/across-protocol/constants/pull/113/commits/5835e64db6bc8d1c86779afdab6b0d402e12cf7a)